index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="./style.css" />
+		<link
+			href="https://fonts.googleapis.com/css?family=Poppins"
+			rel="stylesheet"
+		/>
+		<link
+			rel="stylesheet"
+			href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+		/>
+		<title>My portfolio</title>
+	</head>
+	<body>
+		<section id="header">
+			<div class="logo-placeholder">
+				<p class="logo">SHAH</p>
+			</div>
+			<div class="menu">
+				<span class="material-symbols-outlined"> menu </span>
+			</div>
+		</section>
+		<section id="headline_section">
+			<div class="headline">
+				<div>
+					<h2>I'm Shah Zaman. Glad to see you!</h2>
+				</div>
+				<div>
+					<p>
+						I’m a software developer! I can help you build a product
+						, feature or website Look through some of my work and
+						experience! If you like what you see and have a project
+						you need coded, don’t hesitate to contact me.
+					</p>
+				</div>
+				<div><p id="its_connect">LET’S CONNECT</p></div>
+				<div class="social_icons">
+					<img
+						class="icons"
+						src="./images/Twetter.svg"
+						alt="twetter"
+					/>
+					<img
+						class="icons"
+						src="./images/Linkedin icon.svg"
+						alt="Linkedin"
+					/>
+					<img class="icons" src="./images/madiam.svg" alt="madiam" />
+					<img class="icons" src="./images/github.svg" alt="github" />
+					<img class="icons" src="./images/victor.svg" alt="victor" />
+				</div>
```

Comment:
Kindly I suggest you add The list of icons inside a `ul` tag to group them, and `li` per icon; note that each icon has a link (`a` tag).